### ISSUE TITLE
When LK is not turned off properly, task setting is messed up

### ISSUE THREAD

**Issue by polachp** [LK8000/LK8000#1358]
On last competition, I have made task definition and also saved the task. Then while waiting on the start gate, my phone froze so I have to restart it. After starting LK again, my task was set up to the one we did previous day.

Replication steps:
 - setup or choose task, turn off LK800 properly (day 1 of competition)
 - start lk8000, setup new task for day 2 of competition
 - restart the phone wihtou closing the LK
 - after phone restart open LK8000 - day !1! task is now active. 

If you havent saved your day2 task into file, you have lost it! (once in the air, this is a BIG problem!!!)

Solution:
Save task as current after leaving task definition menu, so even if you restart your phone and dont turn off LK properly, the last task setup is active as expected.
